scripts/mbedtls_framework/psa_compliance.py

```diff
@@ -35,8 +35,10 @@ def test_compliance(library_build_dir: str,
     """Check out and run compliance tests.
 
     library_build_dir: path where our library will be built.
-    psa_arch_tests_ref: tag or sha to use for the arch-tests.
-    patch: patch to apply to the arch-tests with ``patch -p1``.
+    psa_arch_tests_ref: tag or sha to use for the arch-tests
+                        (empty=default/leave alone).
+    patch: patch to apply to the arch-tests with ``patch -p1``
```

Comment:
The parameter is `patch_files` and is a list, so suggesting `patch_files: patches to apply...`